doc/start/hardware-recommendations.rst

```diff
@@ -21,21 +21,25 @@ data cluster (e.g., OpenStack, CloudStack, etc).
 CPU
 ===
 
-CephFS metadata servers are CPU intensive, so they should have significant
-processing power (e.g., quad core or better CPUs) and benefit from higher clock
-rate (frequency in GHz). Ceph OSDs run the :term:`RADOS` service, calculate
-data placement with :term:`CRUSH`, replicate data, and maintain their own copy of the
-cluster map. Therefore, OSD nodes should have a reasonable amount of processing
-power. Requirements vary by use-case; a starting point might be one core per
-OSD for light / archival usage, and two cores per OSD for heavy workloads such
-as RBD volumes attached to VMs.  Monitor / manager nodes do not have heavy CPU
-demands so a modest processor can be chosen for them.  Also consider whether the
-host machine will run CPU-intensive processes in addition to Ceph daemons. For
-example, if your hosts will run computing VMs (e.g., OpenStack Nova), you will
-need to ensure that these other processes leave sufficient processing power for
-Ceph daemons. We recommend running additional CPU-intensive processes on
-separate hosts to avoid resource contention.
-
+CephFS metadata servers are CPU intensive. CephFS metadata servers should
+therefore have quad-core or better CPUs and high clock rates (GHz). OSD nodes
+need hardware with enough processing power to run the RADOS service, calculate
+data placement with CRUSH, replicate data, and maintain their own copies of the
+cluster map.
+
```

Comment:
Suggest:
s/metadata servers/metadata servers (MDS)/

s/therefore..Ghz)/have a high clock frequency (GHz) CPU model. For MDS purposes, higher _frequency_ cores are more beneficial than a large _number_ of cores.  Four or eight cores should be plenty for a server that only hosts the MDS service.  If you must colocate other services, eg. Mon, manager, or OSDs, a larger number of cores is advised.

s/need hardware with/need/

s/map/map.  More cores are beneficial for OSD services; we suggest at least two cores per OSD.  NVMe OSDs may benefit from additional cores, and a higher clock frequency may decrease op latency on these very fast devices as well./